src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -193,6 +194,22 @@ static void setup() {
     Logger.setup();
   }
 
+  @Test
+  @DisplayName("test isVersionNewer")
+  void testVersionComparison() {
+    // Basic checks: version numbers work fine
+    assertFalse(isVersionNewer("1.3.0", "1.3.0"));
+    assertFalse(isVersionNewer("1.2.0", "1.3.0"));
+    assertFalse(isVersionNewer("1.3.0", "1.3.1"));
+    assertTrue(isVersionNewer("1.3.1", "1.3.0"));
+
+    // Version comparison ignores the -alpha specifier
+    assertFalse(isVersionNewer("1.2.0-alpha", "1.3.0"));
+    assertTrue(isVersionNewer("1.4.0-alpha", "1.3.0"));
+    assertFalse(isVersionNewer("1.4.0-alpha", "1.4.0"));
+    assertFalse(isVersionNewer("1.4.0", "1.4.0-alpha"));
```

Comment:
Please also add a check that makes sure:

`assertTrue(isVersionNewer("1.4.0-beta", "1.4.0-alpha"));`

And also curious what happens if a version is just a string like "foo"? Maybe we'd expect that to throw an exception?

Reply:
Alas, it is not true. I've opened an Issue with Jackson: https://github.com/FasterXML/jackson-core/issues/1050

Currently, `alpha` and `beta` are considered the same version.